frontend/src/types.ts

```diff
@@ -2546,6 +2547,11 @@ export interface PathsFilterType extends FilterType {
     path_dropoff_key?: string // Paths People Dropoff Key
 }
 
+export interface CalendarHeatmapFilterType extends FilterType {
+    // Reserved for future filter properties
+    dummy?: string
+}
```

Comment:
Same here, if it doesn't let you keep an empty type for the linter, I think it is best to add a linter exception than to keep `dummy`